sys/include/net/ng_icmpv6.h

```diff
@@ -74,7 +75,7 @@ void ng_icmpv6_demux(kernel_pid_t iface, ng_pktsnip_t *pkt);
  * @return  The ICMPv6 message on success
  * @return  NULL, on failure
  */
-ng_pktsnip_t *ng_icmpv6_build(uint8_t type, uint8_t code, size_t size);
+ng_pktsnip_t *ng_icmpv6_build(ng_pktsnip_t *next, uint8_t type, uint8_t code, size_t size);
 
```

Comment:
Just a matter of taste, but I would expect the payload argument after the arguments for the header fields. If you disagree, I won't object and ACK.